runtime/include/j9memcategories.h

```diff
@@ -52,7 +52,7 @@
 #define J9MEM_CATEGORY_UNUSED10 10 /* #define J9MEM_CATEGORY_JIT 10 */
 #define J9MEM_CATEGORY_UNUSED11 11 /* #define J9MEM_CATEGORY_JIT_CODE_CACHE 11 */
 #define J9MEM_CATEGORY_UNUSED12 12 /* #define J9MEM_CATEGORY_JIT_DATA_CACHE 12 */
-#define J9MEM_CATEGORY_HARMONY 13
+#define J9MEM_CATEGORY_UNUSED13 13 /* #define J9MEM_CATEGORY_HARMONY 13 */
```

Comment:
I don't think there's much value in keeping this comment.

Reply:
I'm following the pattern of the previous categories removed.

Reply:
Got it. I don't expect we need to keep the values the same from release to release.